modules/openapi-generator/src/main/resources/Java/gradle-wrapper.properties.mustache

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
not sure if everyone is ready to use latest gradle 8.x in their projects. we will see.

Reply:
CI is green - which gives me confidence 😎